src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using ScottPlot.Drawing;
+using static System.Net.WebRequestMethods;
+
+// Credits:
+//
+// Inspired (and expanding) by https://github.com/dotnet-ad/Microcharts/blob/main/Sources/Microcharts/Charts/RadialGaugeChart.cs
+//
+// Lighten or darken color
+// https://stackoverflow.com/questions/801406/c-create-a-lighter-darker-color-based-on-a-system-color
+// https://www.pvladov.com/2012/09/make-color-lighter-or-darker.html
+// https://gist.github.com/zihotki/09fc41d52981fb6f93a81ebf20b35cd5
+//
+// Circular Segment
+// https://github.com/falahati/CircularProgressBar/blob/master/CircularProgressBar/CircularProgressBar.cs
+// https://github.com/aalitor/AltoControls/blob/on-development/AltoControls/Controls/Circular%20Progress%20Bar.cs
+//
+// http://csharphelper.com/blog/2015/02/draw-lines-with-custom-end-caps-in-c/
+//
+// Text on path
+// http://csharphelper.com/blog/2018/02/draw-text-on-a-circle-in-c/
+// http://csharphelper.com/blog/2016/01/draw-text-on-a-curve-in-c/
+
+
+// This file should be placed in this address
+// https://github.com/ScottPlot/ScottPlot/tree/master/src/ScottPlot/Plottable
+// and named RadialGaugePlot.cs
+namespace ScottPlot.Plottable
+{
+    /// <summary>
+    /// A radial gauge chart is a graphical method of displaying scalar data in the form of 
+    /// a chart made of circular gauges so that each scalar is represented by each gauge.
+    /// 
+    /// Data is managed using a single array where each element is asigned to each gauge.
+    /// Internally this data is stored in a single array and is converted to angular paramters,
+    /// through ComputeAngularData(), which are more suitable for drawing purposes and stored in a 2D array.
+    /// </summary>
+    public class RadialGaugePlot : IPlottable
+    {
+        #region Properties & fields
+        /// <summary>
+        /// Data to be plotted.
+        /// It's copied from of the data passed to either the constructor or the <see cref="Update(double[])"/> method.
+        /// </summary>
+        protected double[] DataRaw;
+
+        /// <summary>
+        /// Angular data (first column: initial angle; second column: swept angle) computed from <see cref="DataRaw"/>.
+        /// </summary>
+        protected double[,] DataAngular;
+
+        /// <summary>
+        /// The maximum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
+        /// </summary>
+        protected double MaxScale
+        {
+            get => _MaxScale;
+            set
+            {
+                _MaxScale = value;
+                ComputeAngularData();
+            }
+        }
+        private double _MaxScale;
+
+        /// <summary>
+        /// The minimum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
```

Comment:
I think you copy-pasted this comment from above incorrectly

Reply:
I'm afraid I'm failing to see your point.

Reply:
The comment for `MinScale` was the same as the comment for `StartingAngleGauges`. It's resolved now.